postinst

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Enable on boot
+/etc/init.d/wrtbwmon enable
+
+# Configure our cron
+if [ -d "/etc/crontabs" ]; then
+  echo "PATH=/usr/sbin:/usr/bin:/sbin:/bin" > wrtbwmon
```

Comment:
I'm not familiar with postinst scrips. Where does "wrtbwmon" get created? Is it a cron script?